### PR TITLE
#1354 Fix pipeline new page and worker page e2e test fails

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1354 )
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
retry manager

**Comment by wu87988622** [comment]
retry manager

**Comment by wu87988622** [comment]
retry manager

**Comment by wu87988622** [comment]
retry manager

**Comment by konekoya** [comment]
retry manager

File: ohara-manager/client/cypress/integration/pipelineNewPage.test.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ describe('PipelineNewPage', () => {
     });
   });
 
-  it.skip('saves and remove a connector even after page refresh', () => {
+  it('saves and remove a connector even after page refresh', () => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/integration/pipelineNewPage.test.js]: Could you help me change this to _saves and removes_...

File: ohara-manager/client/cypress/integration/pipelineNewPage.test.js [hunk 2]
```diff
@@ -154,7 +154,7 @@ describe('PipelineNewPage', () => {
       .should('not.be.exist');
   });
 
-  it.skip('ftp sink source connect to topic write to graph', () => {
+  it('ftp sink source connect to topic write to graph', () => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/integration/pipelineNewPage.test.js]: connects Ftp soure -> Topic -> Ftp sink

File: ohara-manager/client/cypress/integration/pipelineNewPage.test.js [hunk 3]
```diff
@@ -212,17 +215,17 @@ describe('PipelineNewPage', () => {
...
+          .get(`li[data-value=${topic.name}]`)
+          .click();
       })
       .wait('@putPipeline')
       .get('g.edgePath')
       .should('have.length', 2);
   });
 
-  it.skip('hdfs sink source connect to topic write to graph', () => {
+  it('hdfs sink source connect to topic write to graph', () => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/integration/pipelineNewPage.test.js]: connects Jdbc source -> Topic -> Hdfs sink together

File: ohara-manager/client/cypress/integration/workerPage.test.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ describe('WorkerPage', () => {
     cy.deleteAllWorkers();
   });
 
-  it.skip('creates a new cluster', () => {
+  it('creates a new cluster', () => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/integration/workerPage.test.js]: creates a new connect worker cluster

**Comment by konekoya** [comment]
And enable the homepage test suite...

**Pull request merged**